README.md

```diff
@@ -84,7 +81,7 @@ Slimcoin is a P2P project - there is no “official dev”, nor it is managed by
   - [Source Code](https://github.com/slimcoin-project/Slimcoin)
 
 * Documentation: <add documentation link to slim website>
-  - [SlimCoin Whitepaper](https://slimcoin.info/whitepaperSLM.pdf), 
```

Comment:
This document is in place, the link seems to be the right one. Do you think it would be better to point to the github document here?